### PR TITLE
Feature/winner modal

### PR DESCRIPTION
**Pull request by aliroemhildt**
## Checklist
- [ ] Tests written where required
- [ ] All tests passing
- [ ] Unused code/prys/etc deleted
Testing Coverage at <percentage>%

## Changes Made
  
What (if any) features are you implementing?
- create modal for select winner button that adds double confirmation when choosing a winner

What (if anything) did you refactor?
- n/a

Were there any issues that arose?
- no, all queries seem to be working 

Is there anything that you need from your teammate?
- n/a

Any other comments, questions, or concerns?
- we might want to add some additional feedback after the winner is selected in the modal

### CODE REVIEW

File: src/components/Modal/Modal.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+const Modal = ({ name, setShowModal, id }) => {
+  const [selectWinner, { loading, error, data }] = useMutation(UPDATE_WINNER, {
+    variables: { id: id, winner: true }
+  })
+
+  const updateWinner = (event) => {
+    selectWinner()
+      .then(response => console.log(response.data.updateWinner.submission))
+      .then(() => closeModal())
+  }
```
Comment (mr-ryan12) [inline src/components/Modal/Modal.js:28]: Thank you for creating this Modal. What a great idea!

File: src/components/SubmissionDetailsPage/SubmissionDetailsPage.js [hunk 2]
```diff
@@ -18,46 +20,56 @@ const GET_SUBMISSION = gql`
...
+  // })
 
-  const handleClick = (event) => {
-    selectWinner()
-      .then(response => console.log(response.data.updateWinner.submission))
-  }
+  // const handleClick = (event) => {
+  //   selectWinner()
+  //     .then(response => console.log(response.data.updateWinner.submission))
+  // }
```
Comment (mr-ryan12) [inline src/components/SubmissionDetailsPage/SubmissionDetailsPage.js:55]: Do we need this?

File: src/components/SubmissionDetailsPage/SubmissionDetailsPage.js [hunk 3]
```diff
@@ -18,46 +20,56 @@ const GET_SUBMISSION = gql`
...
+//         id
+//         name
+//         email
+//         genre
+//         songTitle
+//         winner
+//       }
+//     }
+//   }
+// `
```
Comment (mr-ryan12) [inline src/components/SubmissionDetailsPage/SubmissionDetailsPage.js:36]: Do we need this?

File: src/components/Modal/Modal.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+        email
+        genre
+        songTitle
+        winner
+      }
+    }
+  }
+`
+
+const Modal = ({ name, setShowModal, id }) => {
```
Comment (oacortez) [inline src/components/Modal/Modal.js:19]: Thank you for creating this modal component Ali! Looks great!

**Review by oacortez — APPROVED** [review]
Thank you for creating this branch Ali! This is a great feature to show the admin to double make sure they will be selecting a winner. A modal is just perfect! Thanks once again!

**Review by mr-ryan12 — APPROVED** [review]
Thank you so much Ali! This looks great!

**Review by Echory — APPROVED** [review]
Thanks for doing the modal ! There are some comments to delete but we can go through that later!

**Pull request merged**